Source/Krypton Toolkit/Shared/Krypton.Toolkit.Suite.Extended.Tools/Classes/General/MissingFrameWorkAPIs.cs

```diff
@@ -49,7 +47,7 @@ public static bool IsNullOrWhiteSpace(string value)
         /// <returns>The values.</returns>
         public static string[] SplitArray(string inputValue, string delimiter, int limit = -1)
         {
-            string[] output = Strings.Split(inputValue, delimiter, limit, CompareMethod.Text);
+            string[] output = Microsoft.VisualBasic.Strings.Split(inputValue, delimiter, limit, Microsoft.VisualBasic.CompareMethod.Text);
```

Comment:
Really not helping, as the project still has it as a Referenced dll.

Reply:
Just remove the call to "SplitArray" and replace with the string.Split native call

Reply:
The line 379 is in this file 
[ColourTable.txt](https://github.com/Krypton-Suite/Extended-Toolkit/files/6527787/ColourTable.txt)

Might be possible to do it with the split function